### PR TITLE
Issue 66

### LINKED ISSUE #66: Update orgchart library dependency to latest one
Currently, Orgchart Add-on is relying on a very old version of [OrgChart library](https://github.com/dabeng/OrgChart), 2.1.4. Checking the library repository, the latest at this moment is [3.6.0](https://github.com/dabeng/OrgChart/releases). There a few updates regarding not longer depending on the use of font-awesome for icons, some changes on the display of export button, some refactoring, dependencies updates and more. 

It will be nice to make the add-on rely on a newer version and also to check if it could help to fix some identified issues this add-on is currenty presenting/causing (#65, https://github.com/FlowingCode/AddonsDemo/issues/103, https://github.com/FlowingCode/OrgChartAddon/issues/72).

### PR DESCRIPTION
**Pull request by paodb**
Close #66 

Library dependency was updated to latest version 3.7.0. Previous version used was 2.1.4, a really old one. Library had several updates, an a major version update so it was decided the add-on update to be considered as a major update as well. So new version will be 5.0.0.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -290,24 +287,7 @@
                             </execution>
                         </executions>
                     </plugin>
-                    <plugin>
```
Comment (javier-godoy) [inline pom.xml:293]: Why is this plugin removed?

File: pom.xml [hunk 2]
```diff
@@ -192,10 +192,7 @@
 				  <licenseName>apache_v2</licenseName>
 				  <addJavaLicenseAfterPackage>false</addJavaLicenseAfterPackage>
 				  <excludes>
-					<exclude>**/font-awesome.css</exclude>
```
Comment (javier-godoy) [inline pom.xml:195]: Why is this exclusion removed?

**Review by javier-godoy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (paodb) [reply pom.xml:195]: It was already exluded. It is used in demo. It's a font-awesome file that should not be target by the licence plugin.

File: pom.xml (hunk 1 above)
Comment (paodb) [reply pom.xml:293]: That was a mistake. Let me undo this.

Comment (paodb) [reply pom.xml:293]: fixed

File: pom.xml (hunk 2 above)
Comment (javier-godoy) [reply pom.xml:195]: > Why is this exclusion removed.
It wasn't 😂

**Review by javier-godoy — APPROVED** [review]

**Pull request merged**